### ISSUE TITLE
how to cancel the the job

### ISSUE THREAD

**Issue by joffychim** [segmentio/nightmare#672]

**Comment by rosshinkley** [comment]
Could you provide a sample of what you're trying to do?

**Comment by joffychim** [comment]
as the subject, i may cancel the automation when it is typing and kill the process

**Comment by rosshinkley** [comment]
Hm, you can end the process with `.end()`, but not until after other actions in the chain have finished.  Are you asking if you can interrupt the chain to kill the process?  That would not be advisable.  You _could_ kill the Electron process manually using `nightmare.proc.kill()`, but that will put the Nightmare instance into a state where the behavior is not well defined.

**Comment by rosshinkley** [comment]
Closing for lack of activity/information.  If this is still a problem, feel free to reopen/submit a new issue.
